
We need to have a wheel group which has some system privileges to consult the
systemd journal or manage printers with cups.

Upstream says the group does not exist by default.

Upstream-Status: Inappropriate [enable feature]

Signed-off-by: Louis Rannou <lrannou@baylibre.com>
Index: base-passwd-3.5.26/group.master
===================================================================
--- base-passwd-3.5.29.orig/group.master
+++ base-passwd-3.5.29/group.master
@@ -38,5 +38,6 @@
 staff:*:50:
 games:*:60:
 shutdown:*:70:
+wheel:*:80:
 users:*:100:
 nogroup:*:65534:
